libkbfs/folder_block_ops.go

```diff
@@ -2551,12 +2566,14 @@ func (fbo *folderBlockOps) SearchForNodes(ctx context.Context,
 		return nodeMap, nil
 	}
 
-	// Start with the root node
 	rootPtr := md.data.Dir.BlockPointer
 	var node Node
 	if cache == fbo.nodeCache {
```

Comment:
Good idea, I've pasted most of the above comment above the search retry logic.  Not sure if that's the best place, but put it there for now so we don't lose track of it.